README.md

```diff
@@ -19,8 +18,21 @@ what we've done with Openlayers and Cesium but want your own UI? You can do that
 
 ## Getting Started
 
-The build will not run natively in Windows. Perhaps try Cygwin or Linux via a VM or Docker container. 
-It should run great on OS X and typical Linux distributions.
+The build will not run natively in Windows. Perhaps try Cygwin or Linux via a VM or Docker container. It should run great on OS X and typical Linux distributions.
+
+### Yarn
+
+[Yarn](https://yarnpkg.com/en/) is recommended for dependency management over npm, particularly when using plugins and configuration projects. It will automatically link related projects and dependencies in your workspace.
+
+* Clone [opensphere-yarn-workspace](https://github.com/ngageoint/opensphere-yarn-workspace)
+* Clone OpenSphere to the `workspace` directory
+* `yarn install`
+* `npm run build`
```

Comment:
If we are recommending `yarn`, then why not use it to run scripts as well? `yarn run build`

Reply:
Either works for me. We're assuming npm is installed and those two commands are a horse apiece, so you can run either one. All of our scripts use `npm run`, so I went with that.